app/src/main/java/com/spascoding/taskycourse/feature_agenda/data/local/entities/Event.kt

```diff
@@ -13,4 +15,5 @@ data class Event(
     val remindAt: Long,
     val host: String,
     val isUserEventCreator: Boolean,
+    val photos: List<String>?
```

Comment:
If you decided to save this as JSON, wouldn't this just be a string rather than a list of strings?